test/end-to-end/test-end-to-end-dist.js

```diff
@@ -10,9 +10,6 @@ fs.accessSync(path.join(distDir, 'index.cjs.js'));
 fs.accessSync(path.join(distDir, 'index.js'));
 
 const customElementsDir = path.join(distDir, 'custom-elements');
```

Comment:
Does this variable for `/dist/custom-elements` still get used? If not, can we rm it?